### ISSUE TITLE
FEATURE: Local video chat application

### ISSUE THREAD

**Issue by krishhteja** [hiteshchoudhary/apihub#215]
### Describe the Feature

Build a local video chat application using webRTC

### Use Case

This helps people learn more about building applications that can support video calls

**Comment by vikaskr-07** [comment]
can I use webRTC for video call and webRTC for chat option

**Comment by zaid-patel** [comment]
Hello, I want to work on this issue.

**Comment by krishhteja** [comment]
@vikaskr-07 Yes. I've built a similar application during covid. I'll push it here by this weekend and @zaid-patel we can together make it better

**Comment by krishhteja** [comment]
@vikaskr-07 [Here's the example](https://github.com/krishhteja/videochat) I was talking about. @zaid-patel if you wish to, you can use this as base and improve it.

**Comment by wajeshubham** [comment]
@krishhteja Could you please give some overview on what all features are we covering and how this can be backend only module which is easy and feasible to work with any client (web, mobile, desktop etc)? Also, What are we planning to use to exchange ice candidates between peers?

**Comment by krishhteja** [comment]
@wajeshubham The idea of this is not a backend only change. It's going to involve a landing page / video screen in front-end too.

The idea is to use [simple-peer](https://www.npmjs.com/package/simple-peer) and build a small demo application
